client/my-sites/patterns/components/pattern-library/index.tsx

```diff
@@ -176,6 +176,56 @@ export const PatternLibrary = ( {
 
 	const isHomePage = ! category && ! searchTerm;
 
+	const PATTERN_SEO_CONTENT: Record< Category[ 'name' ], { title: string } > = useMemo(
```

Comment:
100% open to feedback on whether or not this is the best way to type this. TS was a little cranky at first and this is where I landed after experimenting a bit.